src/hook/useStepper.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { List as ImmutableList, OrderedMap } from 'immutable'
+import type { StepId, StepStatus } from 'ui/index'
+import type { StepperState } from './useStepper'
+import { useStepper } from './useStepper'
+import { act } from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks'
+
+describe('Considering the stepperReducer function', () => {
+  const previousClicked = 'previousClicked'
+  const stepCompleted = 'stepCompleted'
+  const stepFailed = 'stepFailed'
+  const stepperSubmitted = 'stepperSubmitted'
+  const stepperReseted = 'stepperReseted'
+
+  const emptyState: StepperState = {
+    currentStep: '',
+    enabledSteps: ImmutableList(),
+    stepStatus: OrderedMap()
```

Comment:
For me the name `stepStatus` refers to a status of a single step, but when I see `OrderedMap` I think it's not just a single one.